go.sum

```diff
@@ -118,6 +118,8 @@ github.com/spf13/jwalterweatherman v1.0.0 h1:XHEdyB+EcvlqZamSM4ZOMGlc93t6AcsBEu9
 github.com/spf13/jwalterweatherman v1.0.0/go.mod h1:cQK4TGJAtQXfYWX+Ddv3mKDzgVb68N+wFjFa4jdeBTo=
 github.com/spf13/pflag v1.0.3 h1:zPAT6CGy6wXeQ7NtTnaTerfKOsV6V6F8agHXFiazDkg=
 github.com/spf13/pflag v1.0.3/go.mod h1:DYY7MBk1bdzusC3SYhjObp+wFpr4gzcvqqNjLnInEg4=
+github.com/spf13/pflag v1.0.4 h1:iDFxJFdWvBR5ieANJFvx+fxUFC0lheY/+K/lV73QbEs=
+github.com/spf13/pflag v1.0.4/go.mod h1:5F7vF+0t+QPgqofPFTZZG2xnxtA/FGYmPu6WbdsXIvo=
```

Comment:
nit: probably could run `go mod tidy`

Reply:
this is after having run go mod tidy... I am not sure why there are two versions listed here... perhaps an indirect dependency?